### DISCUSSION TITLE
Verschiedene CMT2300A Module SMD / PIN Typen

### DISCUSSION THREAD

**Discussion by Agerter** [tbnobody/OpenDTU#1251] [General]
Hallo , mal ne Frag an die Experten für die Hoymiles HMS/T Serie wird ja der CMT2300A Funkchip benötigt. Jetzt jetzt hat dieser ja so mini pads und ein Rastermass das man sich schwer tut mit dem Löten.
Daher die Frage kann ich anstatt dem E49 900M20S (SMD)mit E49 900T20D (PINS)verwenden???
Dieser hat Laut Specs eine Frequenz von 850-930MHZ
Wo liegt der unterschied zwischen der Version mit M und der mit T??

Danke schon mal im Vorraus

**Comment by stefan123t** [comment]
S/D am Ende der EByte Modellnummer gibt den Antennen Anschluss an. Das M/T in der Mitte die Bauform des Moduls. Ich bin mir jetzt aber nicht sicher ob das T Modell auch einen SPI anbietet ?
900 ist die Frequenz und 20 dBa ist die in DE/EU zulässige Sendestärke.

Für den E49 900M20S gibt es u.a. bei blinkyparts ein Breakout Board mit dem man den selben Pinout wie ein NRF24 Modul hat.
https://shop.blinkyparts.com/en/Ebyte-Funkmodul-CMT2300A-868-915MHz-Breakoutboard/blink238542

**Comment by Egge72** [comment]
Danke für die schnelle Antwort. .ja das 900T20D hatte halt Pins und man spart sich das löten

**Reply by beppo-dd** [reply]
Hallo,
ich hatte zuerst die User Manuals von [E49-900_T_20S](https://www.cdebyte.com/pdf-down.aspx?id=1133) und [E49-900_T_20D](https://www.cdebyte.com/pdf-down.aspx?id=1324) verglichen und obwohl sie etwas anders aussehen, haben Sie bzgl. des Inhalts kaum Unterschiede:

- min. Versorgungsspannung: 2.3 V  /  2.6 V
- Package-Anschluss: SMD, 1.27 mm / Plug-In, 2.54 mm
- Abmessungen: 26 x 16 mm  /  36 x 21 mm
- Antennen-Anschluss: IPEX/Stamp hole  /  SMA

**Jedoch**: Das *T* in E49-900*T*20S bzw. E49-900*T*20D bedeudet, dass es _kein_ SPI-Interface sondern ein UART-Interface gibt. Also, musst du einen E49-900*M* verwenden! Leider gibt es davon nur die SMD-Variante, siehe [ebyte: Suche nach E49-900M](https://www.cdebyte.com/product_serch/E49-900M):

- [E49-900M20S](https://www.cdebyte.com/products/E49-900M20S)

**Reply by beppo-dd** [reply]
Falls du ein CMT2300A E49-900M20S und eine passend [openDTU Platine](https://github.com/tbnobody/OpenDTU/discussions/1016) verwenden möchtest, kannst du diese bei [DcFamas](https://github.com/DcFamas) unter [openDTU Platine](https://github.com/tbnobody/OpenDTU/discussions/1016) bestellen.

**Comment by Egge72** [comment]
Vielen Dank für die Antwort 😚

beppo-dd ***@***.***> schrieb am Do., 24. Aug. 2023, 05:02:
